app/views/layouts/application.html.erb

```diff
@@ -201,6 +201,7 @@
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Notifications", 'pipelines/notifications' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Emojis", 'pipelines/emojis' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Example Pipelines", 'pipelines/example-pipelines' %></li>
+                  <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Managing Pipelines", 'pipelines/managing-pipelines' %></li>
```

Comment:
What do you think about "Archiving and deleting PIpelines" here? Managing makes me think of organizing them.